components/ratgdo/ratgdo.h

```diff
@@ -51,16 +51,29 @@ namespace ratgdo {
         const uint32_t DOOR_OPEN = 1;
         const uint32_t DOOR_TOGGLE = 2;
         const uint32_t DOOR_STOP = 3;
+
+        const uint32_t TTC_GET_DURATION= 1 ;
+        const uint32_t TTC_0_SEC = 0x000001;
+        const uint32_t TTC_1_SEC = 0x010001;
+        const uint32_t TTC_1_MIN = 0x3c0001;
+        const uint32_t TTC_5_MIN = 0x2c0101;
+        const uint32_t TTC_10_MIN = 0x580201;
```

Comment:
No need for all these constants, the data filed is derived from TTC seconds:
```
uint32_t data = (seconds & 0xff) << 16 | (seconds & 0xff00) | 0x01;
```

Reply:
I can update for calculated data value.  Those were kind of a carry over from the test code.  The `set_ttc_XXXX()` functions will be consolidated as well.